CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+0.7.6
+----
+
+New features:
+
+* [#86](https://github.com/cb372/scalacache/pull/86) adds support for custom serialisation. Thanks to @lloydmeta
```

Comment:
That is some deep existential stuff right there.

Or just blind copy pasting of existing stuff. :laughing: